pkg/util/azureerrors/error.go

```diff
@@ -149,3 +149,11 @@ func ResourceGroupNotFound(err error) bool {
 	}
 	return false
 }
+
+func IsInvalidIdentityUserError(err error) bool {
```

Comment:
n.b. this is poorly named - perhaps `Is4xxError()` - any `azcore.ResponseError` from any call can be passed here. Only in one specific context of this exported function being used will it have anything to do with identities.